ledger/stores/chunked_file_store.py

```diff
@@ -23,16 +25,27 @@ class ChunkedFileStore(FileStore):
 
     firstChunkIndex = 1
 
-    @staticmethod
-    def _fileNameToChunkIndex(fileName):
+    def _fileNameToChunkIndex(self, fileName) -> int:
         try:
-            return int(fileName)
+            path, fileName = os.path.split(fileName)
+
+            path = int(path)
+            fileName = int(os.path.splitext(fileName)[0])
+
+            path *= chunksPerDir
+            return path + fileName
         except:
             return None
 
     @staticmethod
     def _chunkIndexToFileName(index):
-        return str(index)
```

Comment:
Please add a doc with an example of input fileName. Consider using contracts (design by contract) for checking of input here.